pkg/database/migrations.go

```diff
@@ -1929,6 +1929,17 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 					`ALTER TABLE authorized_app_stats DROP COLUMN IF EXISTS tokens_invalid`)
 			},
 		},
+		{
+			ID: "00083-AddTwilioMessageSid",
+			Migrate: func(tx *gorm.DB) error {
+				return multiExec(tx,
+					`ALTER TABLE sms_configs ADD COLUMN IF NOT EXISTS twilio_messaging_service_sid varchar(34)`)
```

Comment:
We should let this be TEXT or varchar(255)

Reply:
done